core/src/main/scala/org/apache/spark/scheduler/SchedulerBackend.scala

```diff
@@ -27,6 +27,9 @@ private[spark] trait SchedulerBackend {
 
   def start(): Unit
   def stop(): Unit
+  /**
```

Comment:
Unrelated?

Reply:
Same, I was reading code and found this comment to be helpful.